azure-pipelines.yml

```diff
@@ -235,7 +235,9 @@ stages:
             -configuration $(_BuildConfig)
             -prepareMachine
             -test
-            -integrationTest
+            # We need to disable integration tests until integration machines update to 17.5 P2:
```

Comment:
Should this block of comments move above the 'script' step, so that it the /p: parameter below is correctly recognized?